n3fit/src/n3fit/performfit.py

```diff
@@ -193,7 +193,8 @@ def performfit(
             max_cores=maxcores,
             model_file=load,
             sum_rules=sum_rules,
-            parallel_models=n_models
+            parallel_models=n_models,
+            replica_id=replica_idxs[0]
```

Comment:
Ok, this convinces me that the `replica` needs to be passed to the `Photon` at initialization time such that what you pass here is just the `Photon`, which will know when to produce the results for one replica or for many.

Otherwise the QED fit won't ever work in parallel!

Reply:
Is the replica information not available at run time? I mean, during NN initialization, when you're also computing the photon.
Once the photon is computed, you just have an array of values, so you need a photon value (of type `np.ndarray`), not a photon instance (of type `Photon`)